kotlinx-coroutines-core/common/src/CancellableContinuation.kt

```diff
@@ -45,18 +111,23 @@ import kotlin.coroutines.intrinsics.*
 @SubclassOptInRequired(InternalForInheritanceCoroutinesApi::class)
 public interface CancellableContinuation<in T> : Continuation<T> {
     /**
-     * Returns `true` when this continuation is active -- it has not completed or cancelled yet.
+     * Returns `true` when this continuation is active -- it was created,
+     * but not yet [resumed][Continuation.resumeWith] or [cancelled][CancellableContinuation.cancel].
+     *
+     * This state implies that [isCompleted] and [isCancelled] are `false`.
      */
     public val isActive: Boolean
 
     /**
-     * Returns `true` when this continuation has completed for any reason. A cancelled continuation
-     * is also considered complete.
+     * Returns `true` when this continuation was completed -- [resumed][Continuation.resumeWith] or
+     * [cancelled][CancellableContinuation.cancel].
+     *
+     * This state implies that [isActive] is `false`.
      */
     public val isCompleted: Boolean
 
     /**
-     * Returns `true` if this continuation was [cancelled][cancel].
+     * Returns `true` if this continuation was [cancelled][CancellableContinuation.cancel].
```

Comment:
Why is this needed? Do we need to introduce such a fix throughout the documentation of every class/interface referring to its other members? I hope not: if just writing `[cancel]` is problematic, to me, this looks like a bug in our documentation tooling.

Reply:
I wanted to be more explicit, I can revert that if you believe it's important.
I.e. it could've been some top-level `cancel`, too much of a context to process at once

Reply:
If it's just a stylistic choice (that no one except the ones reading pure KDoc text will see?), it's not important to me either way, but if this is needed for the link to correctly resolve, that's alarming.